src/CattyTests/Bricks/GlideToBrickTests.swift

```diff
@@ -37,4 +37,40 @@ final class GlideToBrickTests: XCTestCase {
         XCTAssertEqual(brick.xDestination, brick.formula(forLineNumber: 1, andParameterNumber: 0))
         XCTAssertEqual(brick.yDestination, brick.formula(forLineNumber: 1, andParameterNumber: 1))
     }
+
+    func testMutableCopy() {
+        let brick = GlideToBrick()
+
+        brick.durationInSeconds = Formula(double: 1)
+        brick.xDestination = Formula(double: 1)
+        brick.yDestination = Formula(double: 1)
+
+        let copiedBrick: GlideToBrick = brick.mutableCopy(with: CBMutableCopyContext(), andErrorReporting: true) as! GlideToBrick
+
+        XCTAssertTrue(brick.isEqual(to: copiedBrick), "GlideToBricks are not equal")
+        XCTAssertTrue(brick.durationInSeconds.isEqual(to: copiedBrick.durationInSeconds), "durationInSeconds properties are equal")
+        XCTAssertTrue(brick.xDestination.isEqual(to: copiedBrick.xDestination), "xDestination properties are equal")
+        XCTAssertTrue(brick.yDestination.isEqual(to: copiedBrick.yDestination), "yDestination properties are equal")
+    }
+
+    func testMutableCopyForBool() {
```

Comment:
The _isAnimated_ and _isAnimatedInsertBrick_ properties are some inherited burdens which actually should not be publicly visible. They need some major refactoring, but that will be handled in the future. Since the copying of these properties are handled in other tests (e.g. _BrickMutableCopyTests_), there is no need to retest this functionality (I would also suggest to not separately document it in every test). Please remove these tests in your new Brick test classes.